### PR TITLE
Code 139 cleanup deadcode

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
## JIRA
https://osi-cwds.atlassian.net/browse/CODE-139

## Description
Cleanup dead code ClientFormValidator.js and ClientAddEditForm.js from CANS to resolve code smells in Code climate.

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

No tests as the PR is mostly cleaning up dead code.

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

File: app/javascript/Application/components/Client/style.sass [hunk 1]
```diff
@@ -75,29 +75,3 @@
...
-  font-weight: bold
-  text-align: right
-  padding-right: 1rem
-
-.add-edit-form-footer
-  margin: 4rem 5rem 0.5rem 1rem
-
-.add-edit-form-controls
-  display: flex
-  justify-content: flex-end
```
Comment (NagaPradeepGajula-osi) [inline app/javascript/Application/components/Client/style.sass]: I can leave these back if the team feels they can be used some where else.

File: app/javascript/Application/util/constants.js [hunk 2]
```diff
@@ -14,8 +14,6 @@ export const navigation = Object.freeze({
 
   CHILD_LIST: 'CHILD_LIST',
   CHILD_PROFILE: 'CHILD_PROFILE_OVERALL',
-  CHILD_PROFILE_EDIT: 'CHILD_PROFILE_EDIT',
-  CHILD_PROFILE_ADD: 'CHILD_PROFILE_ADD',
```
Comment (AK56) [inline app/javascript/Application/util/constants.js]: I believe we are referring these constants in BreadCrumbHelper.js, probably should yank them out of there too.

Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/util/constants.js]: yeah seems like let me get rid of them too.

Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/util/constants.js]: done

File: app/javascript/Application/components/Client/style.sass (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/components/Client/style.sass]: 👍 wanted to confirm as I have less context

File: app/javascript/Application/util/constants.js (hunk 2 above)
Comment (zliuwd) [reply app/javascript/Application/util/constants.js]: yeah, before that we have client profile editing functionalities, now we don't

**Pull request merged**